recipes/bioconductor-singlecellexperiment/meta.yaml

```diff
@@ -10,22 +10,22 @@ source:
     - 'http://bioconductor.org/packages/{{ bioc }}/bioc/src/contrib/{{ name }}_{{ version }}.tar.gz'
     - 'https://bioarchive.galaxyproject.org/{{ name }}_{{ version }}.tar.gz'
     - 'https://depot.galaxyproject.org/software/bioconductor-{{ name|lower }}/bioconductor-{{ name|lower }}_{{ version }}_src_all.tar.gz'
-  sha256: 6a5ea60b8a9dc442cf1dd9650803677711e6cac6ac6484cd461b14a12264e357
+  sha256: b20ded9e4c32d5411e33bb53d933fb9d4eb68d482fc82923c586d82cdbc924a7
 build:
   number: 0
   rpaths:
     - lib/R/lib/
     - lib/
 requirements:
   host:
-    - 'bioconductor-biocgenerics >=0.26.0,<0.28.0'
-    - 'bioconductor-s4vectors >=0.18.3,<0.20.0'
-    - 'bioconductor-summarizedexperiment >=1.10.1,<1.12.0'
+    - bioconductor-biocgenerics
```

Comment:
the pinning should reflect the bioc release. So in this case `>=0.28.0,<0.30.0` and so on...